scripts/install2

```diff
@@ -56,15 +60,15 @@ function askModule {
     else
         echo -e "\e[1;31m$2 will be skipped\e[0m"
     fi
-    eval $1=\$res
-    return="$res"
+    eval "$1=\$res"
+    # return="$res"
```

Comment:
Why comment out?

Reply:
return does not work in bash (as far as i know) so its a usless line.  Commented it out just in case im mistaken. (so far so good)